.env.sample

```diff
@@ -29,7 +29,7 @@ S3_UPLOAD_BUCKET=""
 S3_UPLOAD_REGION=""
 
 # Misc
-PROCESSING_FEE="31"
+PROCESSING_FEE="33"
```

Comment:
This env variable is barely used, there's still a ton of hardcoding. Not sure why, will try to replace them with the env var in a follow up PR

Client side env variables need to be prefixed with NEXT_PUBLIC_ so that could be why, but I think we can just rename it